tests/unit/lms/security_test.py

```diff
@@ -208,9 +197,8 @@ def test_permits_allow(
         self,
         pyramid_request,
     ):
```

Comment:
```suggestion
        self, pyramid_request):
```

Can't do the suggestion properly... but you get the idea